CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## 0.5.0
+
+- Fixed collision bound calculation in `SafetyModeView`
+- Removed draw distance logic from `SafetyModeView`
+- Added `clean()` method to `SafetyModeView`
+- Fixed `FOREGROUND_SERVICE` permission check for devices with Android < 9
+- `Vision` callbacks moved from `bg` to `UI`
+- Missing `FOREGROUND_SERVICE` permission added
+- Added assets for `UK`
```

Comment:
It's not only assets but separate classifier with assets used. Lets add a note that signs from UK/Europe are classified now.